### ISSUE TITLE
'scss/operator-no-unspaced' does not consider shorthand?

### ISSUE THREAD

**Issue by sigwyg** [stylelint-scss/stylelint-scss#961]
Hello, thank you for all your help.

## Overview

When using a short-hand in the background property with data URIs, it seems to cause an error if the background-image is not at the beginning.

## Issue reproduction
```sample.scss
div {
	// Error
	background: #e6375d url(data:image/png;base64,iVBOR////+LBpLMAA);

	// OK
	background: url(data:image/png;base64,iVBOR////+LBpLMAA);
	background: url(data:image/png;base64,iVBOR////+LBpLMAA) no-repeat top left;
	background: url(data:image/png;base64,iVBOR////+LBpLMAA) #e6375d;
}
```
```.stylelintrc.json
{
  "plugins": ["stylelint-scss"],
  "rules" : {
    "function-url-quotes": "never",
    "scss/operator-no-unspaced": true
  }
}
```
```package.json
{
  "stylelint": "latest",
  "stylelint-scss": "latest"
}
```

### Note

`"function-url-quotes": "never",` is [@wordpress/stylelint-config](https://github.com/WordPress-Coding-Standards/stylelint-config-wordpress/issues/205)

## Expected behavior

Glad to see that [a well-used order](https://developer.mozilla.org/en-US/docs/Web/CSS/Shorthand_properties#background_properties) is allowed.

```css
background: #000 url(images/bg.gif) no-repeat left top;
```

background-color may be first.
[2.10. Backgrounds Shorthand: the background property](https://www.w3.org/TR/css-backgrounds-3/#background)

## perhaps

https://github.com/stylelint-scss/stylelint-scss/blob/e4e8a02f20dd05a132c74a7fc2977a8a914983f4/src/rules/operator-no-unspaced/index.js#L163

I could be wrong, but wouldn't changing `^url\(` to `url\(` fix it?
